packages/garbo/src/fights.ts

```diff
@@ -226,6 +235,11 @@ const secondChainMacro = () =>
           Macro.tryCopier($skill`Digitize`),
         )
           .tryCopier($item`Spooky Putty sheet`)
+          .externalIf(
+            myHp() <
+              Math.min(myMaxhp() * 0.3, get("garbo_restoreHpTarget", 2000)),
+            Macro.tryHaveItem($item`New Age healing crystal`),
+          )
```

Comment:
These can't be externalIfs, unfortunately; we're only evaluating the macro once, before we run the fight, when you're at full HP. Use the existing BALLS predicate!

Reply:
No, because it's still an externalIf

Reply:
Okay; I don't understand BALLS. We're want a regular ass if_ then?